src/mame/includes/avalnche.h

```diff
@@ -8,6 +8,13 @@
 
 #include "sound/discrete.h"
 
+/* Avalanche Discrete Sound Input Nodes */
+#define AVALNCHE_AUD0_EN            NODE_01
+#define AVALNCHE_AUD1_EN            NODE_02
+#define AVALNCHE_AUD2_EN            NODE_03
+#define AVALNCHE_SOUNDLVL_DATA      NODE_04
+#define AVALNCHE_ATTRACT_EN         NODE_05
```

Comment:
Likewise, is there no way to put this inside the `.cpp` file for the audio?  I've been trying to get this stuff out of the headers.